advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,309 @@
+
+![Target Tables Screen](Images/targettablescreen.png)
+
+9. On `Target Server Login Details` Screen, enter EDB Postgres Advanced Server Target Instance user and password and click test connect. Once successful, click Add and then click `Create`.
+
+![Target Server Login Details Screen](Images/targetserverlogin.png)
+
+10. Once done, click `Finish`.
+
+11. To confirm the tables are created on `Target Server`, click the `Target DBMS Server` —> `Tables` —> `Receiving Tables`, the list of the tables will be displayed which are available on `Source DBMS Server`.
+
+![Confirm Tables Are Created on Target Server](Images/confirmtablesontarget.png)
+
+12. Open pgAdmin and connect to EDB Postgres Advanced Server Target Instance, and under Schema `public`, the tables will be created.
+
+![pgAdmin](Images/pgAdmin.png)
+
+13. Mappings of tables setup for replication from EDB Postgres Advanced Server Source Instance to EDB Postgres Advanced Server Target Instance will be created under `Distributions` —> `Mappings`.
+
+![Mappings of Tables](Images/tablemappings.png)
+
+14. Open `Distributions` —> `Newly Created Distribution` —> `Requests` —> `New Request` —> `Copy`. Copy will copy/replicate all the data in tables from EDB Postgres Advanced Server Source Instance to  EDB Postgres Advanced Server Target Instance.
+
+15. Provide required information and click Ok on `Copy Request Properties` Screen.
+
+![Copy Request](Images/copyrequest.png)
+
+## Save the Model
+
+The Model created in the above steps needs to be saved, validated and committed before the Replication can be performed.
+
+1. To save the Model, click `File` and select `Save` option and provide the name for the `Model`.
+
+2. Right Click `Enterprise Data Movement Model` and select `Validate` to validate the Model.
+
+3. No error will be displayed if the Model is valid.
+
+![Model Validation](Images/validmodel.png)
+
+4. Right Click `Enterprise Data Movement Model` and select  `Commit` —> `Full`.
+
+![Model Commit](Images/modelcommit.png)
+
+5. Click Ok on `Commit Model` Screen. This will create the `Model` files.
+
+## Execute Copy Replication using Connect CDC MonCon
+
+Connect CDC MonCon is a GUI application, separate from the Connect CDC Director, that you use for monitoring and control functions. We will use Connect CDC MonCon to execute the Copy Replication. Copy Replication will copy all the data from the Source Tables to the Target Tables.
+
+1. Open Connect CDC MonCon.
+
+2. Click `Model` —> `New Model` from Menu.
+
+3. Provide `Hostname/IP` and select the saved `Model` from the drop down list.
+
+![New Model](Images/newmodel.png)
+
+4. In the `Request` Section, select the `Distribution Model` that was created for copying data created in the above steps and right click and select `start`.
+
+![Distribution Model](Images/distributionmodel.png)
+
+5. Click on `Process` to see the progress.
+
+![Process](Images/Process.png)
```

Comment:
This is another one where it comes across very small. If we really don't need it and can't make it bigger maybe delete this image.

Reply:
removed the image